internal/serverinstall/k8s.go

```diff
@@ -442,7 +442,11 @@ func (i *K8sInstaller) Upgrade(
 
 		log.Info("Pod(s) deleted, k8s will now restart waypoint server ", serverName)
 	} else if waypointStatefulSet.Spec.UpdateStrategy.Type == "RollingUpdate" {
-		log.Info("Update Strategy is 'RollingUpdate', no further action required")
+		log.Info("Update Strategy is 'RollingUpdate'; once the upgrade completes, you may need to restart the pod to update the server image")
+		s.Update("Update Strategy is 'RollingUpdate'; once the upgrade completes, you may need to restart the pod to update the server image")
+		s.Status(terminal.StatusWarn)
+		s.Done()
+		s = sg.Add("")
```

Comment:
I think we can move the `s.Update` checks to further below on `567`: https://github.com/hashicorp/waypoint/blob/4bc862762b6f56bcde7343c596c25c96a370710e/internal/serverinstall/k8s.go#L563-L566

But good call on the Warning, we should update ☝🏻 to be Warn

Reply:
new output
```
✓ Image set to update!
⚠️ Update Strategy is 'RollingUpdate'; once the upgrade completes, you may need to restart the pod to update the server image
✓ Upgrade complete!

Kubernetes is now set to upgrade waypoint server image with its
'RollingUpdate' strategy. This means the pod might not be updated immediately.
```
![image](https://user-images.githubusercontent.com/8461333/126229643-69c3b41b-ba0f-488b-91db-7fe99be8b3cd.png)